blog-posts/event-sourcing-serveless/event-sourcing-serveless.md

```diff
@@ -0,0 +1,49 @@
+---
+published: false
+title: 'The best way to do event sourcing in serverless'
+cover_image:
+description: 'Description of the article'
+tags: serverless, event-sourcing, AWS, CQRS
+series:
+canonical_url:
+---
+
+# Introduction
+You surely know git and it might be the best example of event-sourcing project that every developer know of. It stores every modifications you made in your code into events that are called commits, then the developer has access to the whole history of changes. If you want to implement a project that needs to access the history of what happened throughout the life of the user of even the whole application, then you are in the right place ! My 2 years learnings and the serverless paradigm will help you produce the most stable and the less painful solution you can think of.
```

Comment:
et c'est pas vrai malheureusement, on dirait que git ne stocke que la différence à chaque commit, mais on réalité c'est un snapshot complet qui est utilisé : https://stackoverflow.com/questions/40617288/a-commit-in-git-is-it-a-snapshot-state-image-or-is-it-a-change-diff-patch-delta

Reply:
Il faut que tu remontes le why de pourquoi quelqu'un lirait ton article, c'est quoi le problème que tu résout pour lui ?
C'est un peu gourmand de faire un article à la fois sur ce qu'est l'event-sourcing et le CQRS en plus de proposer une implémentation en serverless. Y'a plétore d'articles qui parle de ces concepts, mais beaucoup moins qui proposent une implémentation en serverless avec le why de chaque service utilisé. Je pense que c'est là que ton article a le plus de valeur.

Un truc qui pourrait être cool, c'est de faire un schéma d'archi technique sans service qui représente un système qui fonctionne en CQRS+event-sourcing, et d'axer ton article sur quels technologies tu vas utiliser pour implémenter chacun des blocs dans une archi serverless AWS